city_scrapers/spiders/chi_southwest_home_equity_i.py

```diff
@@ -0,0 +1,173 @@
+                source=self._parse_source(response),
+            )
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+            yield meeting
+
+    def _parse_title(self, text):
+        name = re.search(r"Southwest Home Equity Assurance Program", text, flags=re.I)
+        minutes = re.search(r"MINUTES", text, flags=re.I)
+        if name and minutes:
+            title = text[name.end() : minutes.start()]
+            return title.replace("\n", "").title().strip()
+        else:
+            return "Board Meeting"
+
+    def parse_pdf(self, url):
+        response = requests.get(url)
+        pdf_text = extract_text(BytesIO(response.content))
+        return pdf_text
+
+    def _parse_start(self, node):
+        # Try to get date from title
+        try:
+            date_str = self.find_date_in_title(node)
+            date_object = datetime.strptime(date_str, "%B %d %Y")
+        except (RuntimeError, ValueError):
+            try:
+                date_str = self.find_date_in_content(node)
+                date_object = datetime.strptime(date_str, "%B %d %Y")
+            except (RuntimeError, ValueError):
+                return None
+
+        try:
+            start_time = self.find_time(node)
+            return date_object + start_time
+
+        except (RuntimeError, ValueError):
+            return date_object + timedelta(hours=18, minutes=30)
+
+    def _parse_links(self, nodes):
+        links = []
+        for node in nodes:
+            if node:
+                links.append(
+                    {"title": self._get_name(node), "href": self._get_link(node)}
+                )
+        return links
+
+    def _get_name(self, node):
+        name = node.xpath("string(.)").get()
+        name = name.replace("(pdf)Download", "")
+        return name
+
+    def _get_link(self, node):
+        url = node.xpath("a/@href").get()
+        if url:
+            return "https:" + url
+        else:
+            return None
+
+    def _parse_source(self, response):
+        return response.url
+
+    def find_date_in_title(self, node):
+        text = self._get_name(node)
+        text_block = text.replace("\n", "")
+        return self._use_date_regex(text_block)
+
+    def find_date_in_content(self, node):
+        content = self.parse_pdf(self._get_link(node))
+        content = content.replace("\n", "")
+        return self._use_date_regex(content)
+
+    def find_time(self, node):
+        content = self.parse_pdf(self._get_link(node))
+        content = content.replace("\n", "")
+        time_str = self._use_time_regex(content)
+        dt = datetime.strptime(time_str, "%I:%M %p")
+        return timedelta(hours=dt.hour, minutes=dt.minute)
+
+    def _use_date_regex(self, text):
+
+        month_ind = re.search(MONTH_REGEX, text, flags=re.I)
+        if month_ind is None:
+            raise RuntimeError("No month found")
```

Comment:
Looks like these are where the `RuntimeError` from above comes. It's generally easier to read if we return `None` instead of an exception, so that would be preferable here if we can also check it in the function using this output